integrations/ultralytics-yolov5/tutorials/sparse-transfer-learning.md

```diff
@@ -0,0 +1,379 @@
+```
+0 0.719010 0.124074 0.022396 0.083333
+0 0.943229 0.133333 0.039583 0.037037
+0 0.787240 0.153241 0.042188 0.041667
+0 0.741667 0.121759 0.017708 0.073148
+0 0.693229 0.100463 0.017708 0.063889
+0 0.670312 0.097222 0.025000 0.075926
+0 0.648177 0.077315 0.022396 0.069444
+0 0.619531 0.050463 0.022396 0.056481
+0 0.492448 0.078704 0.039062 0.059259
+0 0.418229 0.806019 0.034375 0.065741
+0 0.349479 0.646296 0.018750 0.064815
+0 0.461458 0.916204 0.037500 0.052778
+```
+
+Run the following to visualize the data:
+
+```python
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import random, cv2
+from PIL import Image
+
+plt.figure(figsize=(15, 15), facecolor='white')
+
+filename_image = f"aerial-dataset/val/images/00001_frame000000_original.jpg"
+filename_label = filename_image.replace('images','labels').replace('jpg','txt')
+data = pd.read_csv(filename_label, header=None, delimiter=' ', names=["class", "x_center", "y_center", "width", "height"])
+
+print(filename_image)
+print(filename_label)
+im = cv2.imread(filename_image)
+im_size = im.shape[:2]
+for _, bbox in data.iterrows():
+
+  cls, xc, yc, w, h = bbox
+  xmin = xc - w/2
+  ymin = yc - h/2
+  xmax = xc + w/2
+  ymax = yc + h/2
+
+  xmin *= im_size[1]
+  ymin *= im_size[0]
+  xmax *= im_size[1]
+  ymax *= im_size[0]
+
+  start_point = (int(xmin), int(ymin))
+  end_point = (int(xmax), int(ymax))
+  color = (0, 255, 0)
+  thickness = 2
+
+  im = cv2.rectangle(im, start_point, end_point, color, thickness)
+
+plt.axis("off")
+plt.imshow(im)
+```
+
+![sample-image](images/aerial-data-example.png)
+
+#### Create a Config
+
+Save the following configuration file as `aerial-dataset.yaml`:
+
+```yaml
+# aerial-dataset.yaml
+path: /content/aerial-dataset
+train:
+  - train/images
+val:
+  - val/images
+
+# Classes
+nc: 1  # number of classes
+names: ['object']
+```
+
+#### Run Transfer Learning
+
+With the config file setup and data downloaded, we can simply swap in the dataset configuration file in place of the `VOC.yaml`.
+
+```
+sparseml.yolov5.train \
+  --weights zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned75_quant-none?recipe_type=transfer_learn \
+  --recipe zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned75_quant-none?recipe_type=transfer_learn \
+  --recipe_args '{"num_epochs":30}'
+  --data aerial-daraset.yaml.yaml \
+  --patience 0 \
+  --cfg yolov5s.yaml \
+  --hyp hyps/hyp.finetune.yaml
+```
+
+You will notice that we added a `--recipe_args` argument, which updates the transfer 
+learning recipe to run for 30 epochs rather than 55 epochs. While you can always create
+a custom recipe file and pass a local file to script, the `--recipe_args` enables you
+to modify on the fly.
+
+The model achieves ~80% mAP@50.
+
+## Wrapping Up
+
+Checkout the DeepSparse repository for more information on deploying your sparse YOLOv5 models with DeepSparse for GPU class performance on CPUs.
```

Comment:
Check out the DeepSparse repository for more information on deploying your sparse YOLOv5 models with DeepSparse for GPU class performance on CPUs.

Can we link to the repository?